service-discovery/README.adoc

```diff
@@ -31,304 +31,433 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+.Getting the route details for the Quarkus application
+[source,options="nowrap"]
+----
+$ oc get route
+
+NAME                   HOST/PORT
+rhoas-quarkus-kafka    rhoas-quarkus-kafka-jbyrne-dev.apps.sandbox-m2.ll9k.p1.openshiftapps.com
+----
 
-.Verification
-ifdef::qs[]
-* `oc get KafkaConnection` returned created kafka instance
+. On the command line, highlight the URL shown under *HOST/PORT*. Right-click and select *Copy*.
 
-In case of issues you can remove your KafkaConnection by executing
-`oc delete akc name-of-your-kafka` and retry operation
-endif::[]
+. In your web browser, paste the URL for the route.
++
+A web page for the Quarkus application opens.
+
+. In your web browser, append `/prices.html` to the URL.
++
+A new web page entitled *Last price* opens.  Because you haven't yet connected the Quarkus application to your Kafka instance, the price value appears as `N/A`.
 
-[id="proc-create-prices-topic-for-operator_{context}"]
-== Creating the prices Kafka topic in {product}
+[id="proc-creating-topic-in-kafka-instance_{context}"]
+=== Creating a topic in your Kafka instance
 
-For this quick start, the Kafka topic that the Quarkus example application references is called `prices`. You need to create this topic in {product} so that the Quarkus application can interact with it.
+[role="_abstract"]
+In the previous step of this tutorial, you created an example OpenShift application. The application is a Quarkus application that uses a Kafka topic called `prices` to produce and consume messages. In this step, you create the `prices` topic in your Kafka instance so that the Quarkus application can interact with it.
 
 .Prerequisites
-* You've created a Kafka instance in {product} and the instance is in *Ready* state.
+* You've completed the previous steps in this tutorial:
+** xref:proc-verifying-connection-to-openshift-cluster_{context}[]
+** xref:proc-connecting-kafka-instance-to-openshift-cluster_{context}[]
+** xref:proc-deploying-example-application-in-openshift_{context}[]
+* You’ve created a Kafka instance in {product} and the instance is in the *Ready* state. To learn how to create a Kafka instance, see link:{base-url}{getting-started-url}[Getting started with {product-long}].
 
 .Procedure
-. In the *Streams for Apache Kafka* page of the web console, select the name of the Kafka instance that you want to add a topic to.
+. On the link:{service-url}[Kafka Instances] page of the {product} web console, click the name of the Kafka instance that you want to add a topic to.
+
 . Click *Create topic* and follow the guided steps to define the topic details. Click *Next* to complete each step and click *Finish* to complete the setup.
 +
-[.screencapture]
 .Guided steps to define topic details
 image::sak-create-topic.png[Image of wizard to create a topic]
```

Comment:
![image](https://user-images.githubusercontent.com/7964685/127504441-3a027965-1e7a-4c30-a044-9343c7e499e3.png)
maybe a screenshot that also shows the name of the topic that should be used is more direct, I took some times to figure out that the topic name I should be using is specified after the screenshot

Reply:
Good suggestion - thanks.